build/tasks/compare_size.mjs

```diff
@@ -0,0 +1,132 @@
+import chalk from "chalk";
+import fs from "fs";
+import { promisify } from "util";
+import zlib from "node:zlib";
+import { exec as nodeExec } from "child_process";
```

Comment:
If we already rely on modern features here, let's use the `node:` prefix for all core Node modules consistently (there should be an ESLint rule for this!):
```js
import fs from "node:fs";
import { promisify } from "node:util";
import zlib from "node:zlib";
import { exec as nodeExec } from "node:child_process";
```